core/src/main/resources/application.conf

```diff
@@ -84,6 +84,10 @@ system {
 
   // Since the WorkflowLogCopyRouter is initialized in code, this is the number of workers
   number-of-workflow-log-copy-workers = 10
+
+  // Whether to run the metadata summarizer in this Cromwell instance.  There should only be one metadata summarizer
+  // per database, so if this Cromwell instance is being used in read-only mode this value should be `false`.
```

Comment:
@kcibul sorry, I didn't read the ticket thoroughly enough and missed the comment about reusing the refresh interval.  I'm happy to change this, I think that could actually simplify things nicely.